client/blocks/inline-help/popover.jsx

```diff
@@ -412,6 +418,15 @@ function mapStateToProps( state ) {
 	const gutenbergUrl = getGutenbergEditorUrl( state, siteId, postId, postType );
 	const isEligibleForChecklist = isEligibleForDotcomChecklist( state, siteId );
 
+	const isUsingGutenbergPageTemplates =
+		[
+			'modern-business',
+			'professional-business',
+			'sophisticated-business',
+			'calm-business',
```

Comment:
This is missing `elegant-business` and `friendly-business` here and maybe we could also include the `twentynineteen` parent theme?

Reply:
Updated on c94bd2952882e5ff8192ddc06f6ffd58cece2d55.

@mattwiebe any comments on the best way to define _new_ sites?

Reply:
From the issue it seems like the purpose of this is to disable opt-out for MVP users. In that case we could use the `site-builder-mvp` block sticker to detect them (although might not be a future-proof approach in case we decide to start removing it at some point, for example during migration).

Another option we've used is the past is to detect new users based on `user_id` (for the opt-in default editor), so we could use `site_id` here in a similar fashion? (although I'm fine with the creation date too)

Reply:
The `site-builder-mvp` sticker is already outdated. The sites I've created using the improved onboarding do not have it.

Reply:
Thanks for the clarification. I assumed that opting out would completely break only the MVP sites, but now I see that all of the sites prefilled with new onboarding will look wonky too.